src/validationplots/plots.jl

```diff
@@ -51,12 +126,66 @@ end
 """
 # TODO : desc
 """
-function qqplot(fm::MaximumLikelihoodEVA)::Plot
+function qqplot_data(fm::BayesianEVA)::DataFrame
 
-    df = qqplot_data(fm)
+    checkstationarity(fm.model)
 
-    return plot(df, x=:Model, y=:Empirical, Geom.point, Geom.abline(color="red", style=:dash),
-        Guide.xlabel("Model"), Guide.ylabel("Empirical"), Guide.title("Quantile Plot"))
+    y, p = ecdf(fm.model.data.value)
+
+    dist = getdistribution(fm)
+
+    q_sim = Array{Float64}(undef, length(dist), length(y))
+
+    i = 1
+
+    for d in eachslice(dist, dims=1)
+        q_sim[i,:] = quantile.(d, p)
+        i +=1
+    end
+
+    q = vec(mean(q_sim, dims=1))
+
+    return DataFrame(Model = q, Empirical = y)
+
+end
+
+
+"""
+# TODO : desc
+"""
+function returnlevelplot(fm::fittedEVA)::Plot
+
+    if getcovariatenumber(fm.model) > 0
+        @warn "this graphic is not optimized for non-stationary model; plot ignored."
```

Comment:
I feel like this is not consistent with the other functions. In the data functions, when the graph is not optimized for the type of model, we `@info`and then still return the values. It is also true for the `returnlevelplot_data` function. I feel like we sould simply `@info` and still return the plot here for consistency. 
Alternatively, we could `@warn` and not return the data in the data functions.

Reply:
The return level plot makes no sense in the non-stationary context. I would avoid to show it. To make it consistent, changes will be made in returnlevelplot_data

Reply:
Oh ok thanks!